examples/kicker/run_kicker_madx.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022-2023 ImpactX contributors
+# Authors: Ryan Sandberg, Axel Huebl, Chad Mitchell, Marco Garten
```

Comment:
Isn't this copied from the inputs & python equivalent of the kicker, thus?
```suggestion
# Authors: Chad Mitchell, Axel Huebl, Marco Garten
```
?

Reply:
Yep, I copied the license header of `run_kicker.py` and just added my name to it.

Why the change of order? Is that because Chad is the main author of the kicker PR?

Reply:
*Outside GitHub conversation:* it is probably the above, and we just overlooked changing it for `run_kicker.py`.